pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter_test.go

```diff
@@ -43,6 +43,7 @@ import (
 	k8sfake "k8s.io/client-go/kubernetes/fake"
 	"k8s.io/client-go/testing"
 	"k8s.io/client-go/tools/cache"
+	"k8s.io/utils/ptr"
```

Comment:
This cam be avoided. There is already the `"kubevirt.io/kubevirt/pkg/pointer"` package, which can be used as
```
pointer.P(instancetypev1beta1.PreferSpread)
```

Reply:
Updated